3p/environment.js

```diff
@@ -193,7 +193,8 @@ function instrumentEntryPoints(win) {
       next();
       if (typeof fn == 'string') {
         // Handle rare and dangerous string arg case.
-        return (0, win.eval/*NOT OK but whatcha gonna do.*/).call(win, fn);
+        const evil = win.eval;
```

Comment:
@cramforce PTAL at this. should have the same effect without the "suspicious code" which I couldn't seem to suppress with `@suppress` (no matter where i put it).

dist output:

<img width="400" alt="screen shot 2016-09-20 at 2 56 09 am" src="https://cloud.githubusercontent.com/assets/354746/18665862/fab7b684-7edd-11e6-82c0-ec8f8225aa68.png">